tasks/main.yml

```diff
@@ -102,6 +102,22 @@
       {%   endif %}
       {% endfor %}
 
+- name: Configure additional files
+  copy:
+    content: "{{ file['content'] }}"
+    dest: /etc/postfix/{{ file['name'] }}
+    owner: root
+    group: root
+    mode: '0644'
+  loop: "{{ postfix_files }}"
+  no_log: "{{ postfix_no_log }}"
```

Comment:
I suggest making `no_log: true` always and removing `postfix_no_log`, log information here is not much informative anyway, and the role will always run securely this way.

Reply:
Sounds solid, will do :)